docs/source/science/ATL08/ATL08.ipynb

```diff
@@ -0,0 +1,1386 @@
+{
```

Comment:
<p>stop redeclaring the object, streamline so the same object can be used for all the plot examples.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/362/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>+1 to this. You can just use <code>atl08_track</code> since you've declared it above </p><p> </p>